aten/src/ATen/ThreadLocalDebugInfo.cpp

```diff
@@ -30,17 +30,35 @@ void ThreadLocalDebugInfo::_forceCurrentDebugInfo(
   debug_info = info;
 }
 
+/* static */
+void ThreadLocalDebugInfo::_push(
+    DebugInfoKind kind,
+    std::shared_ptr<DebugInfoBase> info) {
+  auto prev_info = debug_info;
+  debug_info = std::make_shared<ThreadLocalDebugInfo>();
+  debug_info->parent_info_ = prev_info;
+  debug_info->kind_ = kind;
+  debug_info->debug_info_ = info;
```

Comment:
Just a naming thing - if you write some_name->some_name it probably means that one of the things needs to be renamed :)